.drone.yml

```diff
@@ -0,0 +1,55 @@
+---
+workspace:
+  base: /drone
+  path: src
+
+
+# Because of drone not building current version of pull request otherwise
+# Suggested by https://github.com/drone/drone/issues/2390#issuecomment-377302134
+clone:
+  git:
+    image: plugins/git:next
+    pull: true
+
+
+pipeline:
+  build:
+    image: docker
+    environment:
+      # Do not use quotes for variable value
+      # because it does not work
+      - DOCKER_BUILD_OPTS=--squash
+      - DOCKER_REPO=meteogroup/python3.7-geolambda
+    volumes:
+      - /var/run/docker.sock:/var/run/docker.sock
+    commands:
+        -  docker build -f Dockerfile.latest -t "$DOCKER_REPO:latest" $DOCKER_BUILD_OPTS .
+        -  docker build -f Dockerfile.1.0.0 -t "$DOCKER_REPO:1.0.0" $DOCKER_BUILD_OPTS .
+
+  run_tests:
+    image: meteogroup/python3.7-geolambda 
+    commands:
+      - gdalinfo ./tests/data/wavewatch_blacksea_6min_st2_gfs_fields_20190808_1200.nc
```

Comment:
Probably good to also test using gdal from python, in order to check that a) python is indeed available and b) gdal pip package is installed and usable.
Oh... just saw that b) is not the case because we only install these later in the connector's dockerfile. Hm... so we have a docker image with python and with gdal but not with the combination of both here. Should we rethink that and already install everything from the requirements*.txt files from https://github.com/developmentseed/geolambda/tree/master/python as well in this docker image. Otherwise it seems a bit counter-intuitive to provide an image where gdal and python can only be used seperately, doesn't it?

Reply:
I would like keep only gdal in requirements not nummpy and rasterio, what do you think ?